client/views/singleOutfit.html

```diff
@@ -13,9 +13,9 @@
     <ul class="commentList" ng-repeat='comment in comments'>
       <li>
```

Comment:
Could it also work by ng-if 'ing the entire <li> to get it all in one fell swoop?